docs/pages/blog.tsx

```diff
@@ -0,0 +1,441 @@
+        </Box>
+      )}
+      <Typography
+        component="h2"
+        fontWeight="bold"
+        variant="subtitle1"
+        sx={{
+          mb: 0.5,
+        }}
+      >
+        <Link
+          aria-describedby={`describe-${props.slug}`}
+          href={`/blog/${props.slug}`}
+          sx={{
+            color: 'text.primary',
+            '&:hover': {
+              textDecoration: 'underline',
+            },
+          }}
+        >
+          {props.title}
+        </Link>
+      </Typography>
+      <Typography color="text.secondary" sx={{ mb: 'auto' }}>
+        {props.description}
+      </Typography>
+      {props.authors && (
+        <AvatarGroup
+          sx={{
+            mt: 2,
+            mb: 1,
+            alignSelf: 'flex-start',
+            '& .MuiAvatar-circular': {
+              width: '28px',
+              height: '28px',
+              border: 3,
+              borderColor: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[800]
+                  : theme.palette.grey[100],
+              backgroundColor: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[700]
+                  : theme.palette.grey[100],
+              color: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[100]
+                  : theme.palette.grey[800],
+              fontSize: (theme) => theme.typography.pxToRem(13),
+              fontWeight: 500,
+            },
+          }}
+        >
+          {(props.authors as Array<keyof typeof AUTHORS>).map((author) => (
+            <Avatar key={author} alt="" src={AUTHORS[author]?.avatar} />
+          ))}
+        </AvatarGroup>
+      )}
+      <Box
+        sx={{
+          display: { sm: 'block', md: 'flex' },
+          justifyContent: 'space-between',
+          alignItems: 'end',
+        }}
+      >
+        <Box sx={{ position: 'relative' }}>
+          {props.authors && (
+            <Typography variant="body2" fontWeight="500">
+              {props.authors
+                .slice(0, 3)
+                .map((userId) => {
+                  const name = AUTHORS[userId as keyof typeof AUTHORS]?.name;
+                  if (name) {
+                    if (props.authors && props.authors.length > 1) {
+                      // display only firstName
+                      return name.split(' ')[0];
+                    }
+                    return name;
+                  }
+                  return userId;
+                })
+                .join(', ')}
+              {props.authors.length > 2 && ', and more.'}
+            </Typography>
+          )}
+          {props.date && (
+            <Typography variant="caption" fontWeight="400" color="text.secondary">
+              {new Date(props.date).toDateString()}
+            </Typography>
+          )}
+        </Box>
+        <Button
+          component="a"
```

Comment:
Is it intended to have no left margin here on hover?

![image](https://user-images.githubusercontent.com/42154031/149058406-8700664e-b199-497a-9723-f4e70f122d2b.png)

Reply:
Fixed, missed that one, thanks!